/**:
  ros__parameters:
    maximum_queue_size: 5
    timeout_sec: 1.0
    is_motion_compensated: true
    publish_synchronized_pointcloud: true
    keep_input_frame_in_synchronized_pointcloud: true
    publish_previous_but_late_pointcloud: false
    synchronized_pointcloud_postfix: pointcloud
    input_twist_topic_type: twist
    input_topics: [
                    "/sensing/lidar/front_lower/pointcloud_raw_ex",
                    "/sensing/lidar/front_upper/pointcloud_raw_ex",
                    "/sensing/lidar/rear_lower/pointcloud_raw_ex",
                    "/sensing/lidar/rear_upper/pointcloud_raw_ex",
                    "/sensing/lidar/right_lower/pointcloud_raw_ex",
                    "/sensing/lidar/right_upper/pointcloud_raw_ex",
                    "/sensing/lidar/left_lower/pointcloud_raw_ex",
                    "/sensing/lidar/left_upper/pointcloud_raw_ex"
                ]
    output_frame: base_link
    lidar_timestamp_offsets: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.01]
    lidar_timestamp_noise_window: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
